src/sql/src/session/vars.rs

```diff
@@ -579,6 +579,16 @@ static ENABLE_WITH_MUTUALLY_RECURSIVE: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+/// Feature flag indicating whether monotonic evaluation of one-shot SELECT queries is enabled.
+static ENABLE_MONOTONIC_ONESHOT_SELECTS: ServerVar<bool> = ServerVar {
```

Comment:
If a view was originally planned with this on, then we turn this off, then we restart, will we run into issues when re-planning the view?

Reply:
This only applies to `SELECT` queries, which are ephemeral and shouldn't leave any trace in the catalog. If we expand similar optimizations to views, it is fine to turn of the flag because it doesn't change the output of a query.